apps/frontend/src/index.css

```
@@ -455,6 +455,42 @@
+}
+
+.ProseMirror table {
+  border-collapse: collapse;
+  table-layout: fixed;
+  width: 100%;
+}
+
+.ProseMirror table td,
+.ProseMirror table th {
+  @apply border border-border px-2 py-1 align-top text-sm;
+  min-width: 4rem;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)